test/e2e/testnet/testnet.go

```diff
@@ -317,23 +331,32 @@ func (t *Testnet) Start() error {
 			genesisNodes = append(genesisNodes, node)
 		}
 	}
+	// start genesis nodes asynchronously
 	for _, node := range genesisNodes {
-		err := node.Start()
+		err := node.Instance.StartWithoutWait()
 		if err != nil {
 			return fmt.Errorf("node %s failed to start: %w", node.Name, err)
 		}
 	}
+	// wait for instances to be running
 	for _, node := range genesisNodes {
-		client, err := node.Client()
+		err := node.Instance.WaitInstanceIsRunning()
 		if err != nil {
-			return fmt.Errorf("failed to initialize node %s: %w", node.Name, err)
+			return fmt.Errorf("node %s failed to start: %w", node.Name, err)
 		}
+	}
```

Comment:
We can start all nodes in an async way and then wait for all nodes to be started.
I contrast to start and wait for them one-by-one.

Reply:
Absolutely, it is very useful!